# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../attr/attr-add-destdir.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./getfattr/Makefile.orig	2012-12-30 19:01:00.826873199 +0100
+++ ./getfattr/Makefile	2012-12-30 19:01:38.895781699 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
 install-dev install-lib:
--- ./setfattr/Makefile.orig	2012-12-30 19:02:09.188477295 +0100
+++ ./setfattr/Makefile	2012-12-30 19:02:33.817032689 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
 install-dev install-lib:
--- ./attr/Makefile.orig	2012-12-30 19:05:44.741475380 +0100
+++ ./attr/Makefile	2012-12-30 19:06:09.042034277 +0100
@@ -29,6 +29,6 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
 install-dev install-lib:
--- ./man/man1/Makefile.orig	2012-12-30 19:09:33.218765338 +0100
+++ ./man/man1/Makefile	2012-12-30 19:10:18.519808149 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install : default
-	$(INSTALL) -m 755 -d $(MAN_DEST)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(MAN_DEST)
 	$(INSTALL_MAN)
 install-dev install-lib:
--- ./man/man2/Makefile.orig	2012-12-30 19:14:06.781090088 +0100
+++ ./man/man2/Makefile	2012-12-30 19:15:05.498446855 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install-dev : default
-	$(INSTALL) -m 755 -d $(MAN_DEST)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(MAN_DEST)
 	$(INSTALL_MAN)
 install-lib:
--- ./man/man3/Makefile.orig	2012-12-30 19:15:13.654618902 +0100
+++ ./man/man3/Makefile	2012-12-30 19:15:31.723038252 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install-dev : default
-	$(INSTALL) -m 755 -d $(MAN_DEST)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(MAN_DEST)
 	$(INSTALL_MAN)
 install-lib:
--- ./man/man5/Makefile.orig	2012-12-30 19:15:39.451286442 +0100
+++ ./man/man5/Makefile	2012-12-30 19:15:55.635594148 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install : default
-	$(INSTALL) -m 755 -d $(MAN_DEST)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(MAN_DEST)
 	$(INSTALL_MAN)
 install-dev install-lib:
--- ./doc/Makefile.orig	2012-12-30 19:21:23.063176559 +0100
+++ ./doc/Makefile	2012-12-30 19:22:25.940631156 +0100
@@ -32,12 +32,12 @@
 
 install: default
 	$(SUBDIRS_MAKERULE)
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 PORTING CHANGES.gz $(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 PORTING CHANGES.gz $(DESTDIR)$(PKG_DOC_DIR)
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -S CHANGES.gz $(PKG_DOC_DIR)/changelog.gz
+	$(INSTALL) -S CHANGES.gz $(DESTDIR)$(PKG_DOC_DIR)/changelog.gz
 else
-	$(INSTALL) -m 644 COPYING $(PKG_DOC_DIR)
+	$(INSTALL) -m 644 COPYING $(DESTDIR)$(PKG_DOC_DIR)
 endif
 
 install-dev install-lib: $(SUBDIRS)
--- ./Makefile.orig	2012-12-30 19:34:25.553318150 +0100
+++ ./Makefile	2012-12-30 19:40:48.606191676 +0100
@@ -90,8 +90,8 @@
 	fi
 
 install: default $(addsuffix -install,$(SUBDIRS))
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 README $(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 README $(DESTDIR)$(PKG_DOC_DIR)
 
 install-dev: default $(addsuffix -install-dev,$(SUBDIRS))
 
--- ./include/Makefile.orig	2012-12-30 19:41:56.779747168 +0100
+++ ./include/Makefile	2012-12-30 19:42:39.492719627 +0100
@@ -33,7 +33,7 @@
 include $(BUILDRULES)
 
 install-dev: default
-	$(INSTALL) -m 755 -d $(PKG_INC_DIR)
-	$(INSTALL) -m 644 $(INST_HFILES) $(PKG_INC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_INC_DIR)
+	$(INSTALL) -m 644 $(INST_HFILES) $(DESTDIR)$(PKG_INC_DIR)
 
 install install-lib:
--- ./include/buildmacros.orig	2010-01-22 23:55:11.000000000 +0100
+++ ./include/buildmacros	2012-12-30 19:48:26.180795088 +0100
@@ -68,9 +68,9 @@
 ifeq ($(ENABLE_SHARED),yes)
 INSTALL_LTLIB = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_LIB_DIR); \
-	../$(INSTALL) -m 644 -T so_dot_version $(LIBNAME).lai $(PKG_LIB_DIR); \
-	../$(INSTALL) -T so_dot_current $(LIBNAME).lai $(PKG_LIB_DIR)
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -m 644 -T so_dot_version $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -T so_dot_current $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR)
 endif
 
 # Libtool thinks the static and shared libs should be in the same dir, so
@@ -82,15 +82,15 @@
 ifeq ($(ENABLE_SHARED),yes)
 INSTALL_LTLIB_DEV = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 $(LIBNAME).lai $(PKG_DEVLIB_DIR)/$(LIBNAME).la ; \
-	../$(INSTALL) -m 755 -d $(PKG_LIB_DIR); \
-	../$(INSTALL) -T so_base $(LIBNAME).lai $(PKG_LIB_DIR); \
-	if test "x$(PKG_DEVLIB_DIR)" != "x$(PKG_LIB_DIR)" ; then \
-	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).a $(PKG_LIB_DIR)/$(LIBNAME).a; \
-	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).la $(PKG_LIB_DIR)/$(LIBNAME).la; \
-	../$(INSTALL) -S $(PKG_LIB_DIR)/$(LIBNAME).so $(PKG_DEVLIB_DIR)/$(LIBNAME).so; \
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR)/$(LIBNAME).la ; \
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -T so_base $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR); \
+	if test "x$(DESTDIR)$(PKG_DEVLIB_DIR)" != "x$(DESTDIR)$(PKG_LIB_DIR)" ; then \
+	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).a $(DESTDIR)$(PKG_LIB_DIR)/$(LIBNAME).a; \
+	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).la $(DESTDIR)$(PKG_LIB_DIR)/$(LIBNAME).la; \
+	../$(INSTALL) -S $(PKG_LIB_DIR)/$(LIBNAME).so $(DESTDIR)$(PKG_DEVLIB_DIR)/$(LIBNAME).so; \
 	fi
 else
 INSTALL_LTLIB_DEV = $(INSTALL_LTLIB_STATIC)
@@ -98,8 +98,8 @@
 
 INSTALL_LTLIB_STATIC = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR)
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR)
 
 INSTALL_MAN = \
 	@for d in $(MAN_PAGES); do \
@@ -111,7 +111,7 @@
 				-e 's/\\\f[0-9]//g' -e 's/  / /g;q'`; \
 		do \
 			[ -z "$$m" -o "$$m" = "\\" ] && continue; \
-			t=$(MAN_DEST)/$$m.$(MAN_SECTION); \
+			t=$(DESTDIR)$(MAN_DEST)/$$m.$(MAN_SECTION); \
 			if $$first; then \
 				if $(HAVE_ZIPPED_MANPAGES); then \
 					$(ZIP) -9 -c $$d > $$d.gz; _sfx=.gz; \
@@ -131,7 +131,7 @@
 INSTALL_LINGUAS = \
 	@for l in $(LINGUAS) ""; do \
 		if test -f "$$l.mo" ; then \
-			ldir=$(PKG_LOCALE_DIR)/$$l/LC_MESSAGES; \
+			ldir=$(DESTDIR)$(PKG_LOCALE_DIR)/$$l/LC_MESSAGES; \
 			$(INSTALL) -m 755 -d $$ldir; \
 			$(INSTALL) -m 644 $$l.mo $$ldir/$(PKG_NAME).mo; \
 		fi; \
